pkg/controller/controller.go

```diff
@@ -294,12 +302,20 @@ func (c *Controller) initActionSetStatus(as *crv1alpha1.ActionSet) {
 	actions := make([]crv1alpha1.ActionStatus, 0, len(as.Spec.Actions))
 	var err error
 	for _, a := range as.Spec.Actions {
-		var actionStatus *crv1alpha1.ActionStatus
-		actionStatus, err = c.initialActionStatus(as.GetNamespace(), a)
+		if a.Blueprint == "" {
+			// TODO: If no blueprint is specified, we should consider a default.
+			err = errors.New("Blueprint not specified")
```

Comment:
Should we call `c.logAndErrorEvent` here?

Reply:
Fixed